### ISSUE TITLE
Add `role` to global attributes

### ISSUE THREAD

**Issue by estelle** [mdn/content#19942]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes

### What specific section or headline is this issue about?

#list_of_global_attributes

### What information was incorrect, unhelpful, or incomplete?

`role` is not there

### What did you expect to see?

A section on `role` between `part` and `slot`
Role is defined in WAI-ARIA, not in WHATWG, which also needs to be added in the spec table.

There is an entire section devoted to the role attribute -- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles -- but on the global attributes page, the only content is "The ARIA [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) attribute and the multiple [aria-*](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes) states and properties, used for ensuring accessibility."

While we shouldn't include the individual aria-* attributes, because that would pollute the page, the `role` attribute should be added, with a mention and link to attributes
